docroot/modules/custom/webform_custom_submissions/travel_services_config.inc

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Note: this is Production!
+ */
+
+global $DC_PROJECT;
```

Comment:
As previously stated, these configs will need to be created in a Drupal 8 config form